src/script/lua_api/l_particleparams.h

```diff
@@ -226,15 +226,18 @@ namespace LuaParticleParams
 
 			// get the effect settings
 			lua_getfield(L, -1, "style");
-			lua_isnil(L,-1) || (readLuaValue(L, field.style), true);
+			if (!lua_isnil(L,-1))
+				readLuaValue(L, field.style);
```

Comment:
where did the `true` argument go?

Reply:
it's not an argument, it's part of the comma operator. It when nowhere before